package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sanitize-html",
-  "version": "2.0.0-rc.2",
+  "version": "2.0.0-rc.3",
```

Comment:
I'm following the pattern here but don't let me stop you from releasing 2.0.0 final.

Reply:
Let's go for the 2.0.0. Unless you want that to be its own PR, which works too.